### DISCUSSION TITLE
blocked account

### DISCUSSION THREAD

**Discussion by maktubz1** [vercel/community#4279] [Account Issue]
### Question

I already sent an email to Sweatte over a week ago and have not received a response.
![Sem título](https://github.com/vercel/community/assets/110111015/64a3f3e5-a9d7-4284-bc48-c33946f5cdc1)

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Reply by maktubz1** [reply]
It's been more than 2 weeks that I've been waiting for support to get in touch and so far nothing, I haven't received a response yet.

**Reply by mdillgen** [reply]
Same issue here on sign up. no verification email. No error message just nothing, really bad!!!!

**Comment by xuannam12** [comment]
3 weeks no response for me! so bad😰!!!  I have never seen a platform with such bad support😉. I think I will find another solution. can't wait any longer.
